packages/calcite-components/src/components/tab/tab.scss

```diff
@@ -1,3 +1,11 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-block-padding: Specifies the padding of the block `default` slot.
```

Comment:
CSS prop name and doc does not match the component they belong to.

Reply:
I meant the 'block' here as y-padding. Is there a better way to name it?

Reply:
Should the be `--calcite-tab-content-padding`? I think that matches other similar var names.

Reply:
All component-specific tokens share the following naming pattern:

`--calcite-<component>-<prop>`.

Interestingly, I couldn't find this in our conventions, so we should add an entry for this.

Reply:
IMO with the name “content-padding” it should apply a full padding rule if overridden, not just block padding.

Reply:
With the ongoing component-token work, should this be `--calcite-tab-space`, `--calcite-tab-content-space`? cc @alisonailea

Reply:
Does this not have the same issue as above, where tab-space or tab-content-space would be expected to override full padding around, not just y-padding?

Reply:
@Elijbet if it is only applied to the content element inside tab then the name should be `--calcite-tab-content-space`

Reply:
This seems like a different approach than other components? IMO it isn’t descriptive for a component level property IMO. Is the “space” referencing space between the tab nav and the tab etc? If not -  Should we change other components that use padding to match this?

Reply:
> It's the spacing around the tab content, right?
Yeah definitely; just meant it’s not a self explanatory name like “padding”.

Reply:
> All component-specific tokens share the following naming pattern:
> 
> `--calcite-<component>-<prop>`.
> 
> Interestingly, I couldn't find this in our conventions, so we should add an entry for this.

I was thinking about this the other day and using the Token Naming Schema as a guide, you could expand the template to something like:

`--calcite-<component>-<stencil-prop?>-<css-property?>-<modifier?>`

Where the `?` denote optional, so you could end up with a var like `--calcite-tile-background-color` which leaves out the stencil prop since that isn't applicable, but you could also have a var called `--calcite-tile-heading-text-color` and `--calcite-tile-heading-text-color-hover` which use more of those elements.

Reply:
Your right I think I need to be more explicit. It is preferable that we create a single `-space` token from which margin and padding are calculated and applied as `-internal-` variables. This should be set on the `:host`. If you are creating a variable that only applied to padding properties then it should end with `-padding`.

Reply:
For the tile example, it’s also possible the hover color could be set by the user on a pseudo class right? Only internal or inaccessible containers would need to be exposed? Close buttons, etc.

Reply:
> It is preferable that we create a single -space token from which margin and padding are calculated and applied as -internal- variables. This should be set on the :host. If you are creating a variable that only applied to padding properties then it should end with -padding.

We should decide on one or the other or set some guidelines to deal with the following scenario:

1. component adds `--calcite-component-content-space` (used internally for both margin/padding)
2. [🇫🇷2 hours later🇫🇷] component needs to support different props for margin and padding, so it adds `--calcite-component-content-space-padding` and `--calcite-component-content-space-margin`

Does `--calcite-component-content-space` need to be deprecated? If so, it means we need to support the 3 CSS props until the next breaking change.

LMK if I misread your suggestion.

Reply:
Regardless, as @Elijbet stated, let's move forward w/ `--calcite-tab-content-padding` for consistency and we can revisit ☝️ soon.

Reply:
Actually, this might have to be `--calcite-tab-content-block-padding` (semi full circle, eh? 😅) since the content padding isn't equal and I don't think we can use a single CSS prop for block and inline padding if their values aren't equal.

Reply:
Got @alisonailea's blessing on ☝️ (via Teams):

> yes. In this case `--calcite-tab-content-block-padding` is probably best